src/plugins/data_source_management/public/components/edit_data_source/components/edit_form/edit_data_source_form.test.tsx

```diff
@@ -380,6 +397,69 @@ describe('With Registered Authentication', () => {
       }
     );
 
-    expect(mockCredentialForm).toHaveBeenCalled();
+    expect(mockedCredentialForm).toHaveBeenCalled();
+  });
+
+  test('should update the form with registered auth type on click save changes', async () => {
+    const mockedCredentialForm = jest.fn();
+    const mockedSubmitHandler = jest.fn();
+    const authMethodToBeTest = {
+      name: 'Some Auth Type',
+      credentialSourceOption: {
+        value: 'Some Auth Type',
+        inputDisplay: 'some input',
+      },
+      credentialForm: mockedCredentialForm,
+      crendentialFormField: {},
+    } as AuthenticationMethod;
+
+    const mockedContext = mockManagementPlugin.createDataSourceManagementContext();
+    mockedContext.authenticationMethodRegistery = new AuthenticationMethodRegistery();
+    mockedContext.authenticationMethodRegistery.registerAuthenticationMethod(authMethodToBeTest);
+
+    component = mount(
+      wrapWithIntl(
+        <EditDataSourceForm
+          existingDataSource={mockDataSourceAttributesWithRegisteredAuth}
+          existingDatasourceNamesList={existingDatasourceNamesList}
+          onDeleteDataSource={jest.fn()}
+          handleSubmit={mockedSubmitHandler}
+          handleTestConnection={jest.fn()}
+          displayToastMessage={jest.fn()}
+        />
+      ),
+      {
+        wrappingComponent: OpenSearchDashboardsContextProvider,
+        wrappingComponentProps: {
+          services: mockedContext,
+        },
+      }
+    );
+
+    await new Promise((resolve) =>
+      setTimeout(() => {
+        updateInputFieldAndBlur(component, descriptionFieldIdentifier, '');
+        expect(
+          // @ts-ignore
```

Comment:
why do we need //@ts-ignore?

Reply:
Thanks for the comment, replied below, will remove it.

Reply:
Good catch, will remove it.